@stellar/freighter-api/src/addToken.ts

```diff
@@ -0,0 +1,23 @@
+import { submitToken } from "@shared/api/external";
+import { FreighterApiError } from "@shared/api/types";
+import { FreighterApiNodeError } from "@shared/api/helpers/extensionMessaging";
+import { isBrowser } from ".";
+
+export const addToken = async (args: {
+  contractId: string;
+  networkPassphrase?: string;
```

Comment:
if you want to keep this param optional (which makes sense; in our API methods, we tend to keep network passphrase optional), I think you'll need some logic [here](https://github.com/stellar/freighter/pull/1815/files#diff-96f7a3637d5d67d1c18eb2b9f7c8312edd3eaa1706e0c0467f922626010c7271R145) to handle what to do if the network passphrase isn't included. In `submitTx`, for ex, we just default it to Mainnet

Reply:
Nice catch, how does this look? https://github.com/stellar/freighter/pull/1815/commits/8e0e204937d76bf13ce0a073113a43bf644c06fe